packages/gluestick/src/autoUpgrade/__tests__/checkForMismatch.test.js

```diff
@@ -0,0 +1,55 @@
+jest.mock('../getSingleEntryFromGenerator.js', () => jest.fn());
```

Comment:
We need to start also adding `/* flow */` into tests, they are ready. I already did it in several of them. The same goes for the other tests on this review.